### PR TITLE
Block editor UI polish

### PR DESCRIPTION
**Pull request by jossmac**
- move some UI around
- align block-editor styles with keystone's native styles (color, spacing, etc.)
- verbose styled-component names to aid understanding
- find and implement icons for the insert menu
- replace "selection-style" toolbar with a more traditional toolbar, that's sticky
- move to absolute offsets for menus, so we can portal (helps with stacking context issues)
- support "Enter" on dynamic-components to create a new paragraph below
- remove stacking context from editor DOM node
- maintain disclosure line on blocks in edit mode, even when not focused
- a little trickery to hide menus when not focused

### CODE REVIEW

File: website/index.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ const apps = [
 	new GraphQLApp(),
 	new AdminUIApp({
 		adminPath: '/admin',
-		authStrategy,
+		// authStrategy,
```
Comment (jossmac) [inline website/index.js]: TODO: Uncomment after WIP

File: website/utils/dynamic-components-view.js [hunk 2]
```diff
@@ -36,27 +41,26 @@ export let Provider = ({ value, children }) => {
 export function Sidebar({ editor }) {
 	let { adminMeta, components } = useContext(Context);
 	let [view] = adminMeta.readViews([components]);
-	const icon = (
```
Comment (jossmac) [inline website/utils/dynamic-components-view.js]: This was **always** rendering an image icon.

File: website/index.js (hunk 1 above)
Comment (MadeByMike) [inline website/index.js]:
```suggestion
		authStrategy,
```

**Comment by MadeByMike** [comment]
@jossmac I removed the comment around the authStrat - are you happy to move this out of WIP?

**Comment by jossmac** [comment]
No. Not happy to move out of WIP. There's lots of things that aren't finished.

**Pull request merged**